src/arith_uint256.h

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
```

Comment:
There's a base_uint_old present at the bottom of this file that's no longer present in upstream. Do we still need it? Can we remove it as well?

Reply:
All those lines have been removed and the code compiles and runs. It appears to have been introduced between vertcoin-core 0.11 and 0.11.1. Its possible bitcoin was the source but since they cut off their archive at 0.14.0, I can;t look back far enough to see when they inserted, and them removed these lines.

0.15.98 is updated with this change.